subprojects/core/src/main/java/org/gradle/api/internal/changedetection/state/InputFileSnapshotNormalizationStrategy.java

```diff
@@ -23,7 +23,7 @@
 import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 
-public enum TaskFilePropertySnapshotNormalizationStrategy implements SnapshotNormalizationStrategy {
+public enum InputFileSnapshotNormalizationStrategy implements SnapshotNormalizationStrategy {
```

Comment:
Thanks. Renaming annotations is always complicated, as we'll need to ask people to annotate their tasks with both the old and the new annotation. Let's do that separately indeed.